web/locales/el.yml

```diff
@@ -20,40 +21,42 @@ el: # <---- change this to your locale code
   Job: Εργασία
   Arguments: Ορίσματα
   Extras: Extras
-  Started: Ξεκίνησαν
+  Started: Ξεκίνησε
   ShowAll: Εμφάνιση Όλων
   CurrentMessagesInQueue: Τρέχουσες εργασίες <span class='title'>%{queue}</span>
   Delete: Διαγραφή
   AddToQueue: Προσθήκη στην στοίβα
-  AreYouSureDeleteJob: Θέλετε να διαγράψετε την εργασία αυτη;
-  AreYouSureDeleteQueue: Θέλετε να διαγράψετε την %{queue} στοίβα?
+  AreYouSureDeleteJob: Θέλετε να διαγράψετε αυτη την εργασία;
+  AreYouSureDeleteQueue: Θέλετε να διαγράψετε την στοίβα %{queue}; Αυτό θα διαγράψει όλες τις εργασίες εντός της στοίβας, θα εμφανιστεί ξανά εάν προωθήσετε περισσότερες εργασίες σε αυτήν στο μέλλον.
```

Comment:
Too long. Keep it short as this displays in a modal dialog.

Reply:
Reverted.

I had changed it, because it was like this on the [english version](https://github.com/mperham/sidekiq/blob/ff7e41924df8c1b64c87a87abda0720382aa667c/web/locales/en.yml#L30).

Reply:
Oh, my apologies. If that's how the English is, then you can directly translate it. I'd bet I'm wrong about the length issue.

Reply:
No worries.

It has been translated it.